ui/src/app/workflows/components/workflows-list/workflows-list.tsx

```diff
@@ -158,6 +159,7 @@ export function WorkflowsList({match, location, history}: RouteComponentProps<an
     }, [namespace, phases.toString(), labels.toString(), pagination.limit, pagination.offset]); // referential equality, so use values, not refs
 
     useCollectEvent('openedWorkflowList');
+    const {t} = useTranslation();
```

Comment:
same as above, this should be before all the `useEffect` with the other variable declarations at the beginning of the component